partner_tlr_update/models/res_partner.py

```diff
@@ -200,11 +200,13 @@ def update_contact_person(self, xml, parent_id):
         if person_id is not None:
             person_id = person_id.text
             children = self.env['res.users'].search([
```

Comment:
We don't care about other children than the first?
If we are only interested in one child then the variable should be called child and not children.